jobs/resource_job_test.go

```diff
@@ -87,6 +87,9 @@ func TestResourceJobCreate(t *testing.T) {
 							PauseStatus:          "PAUSED",
 						},
 						Queue: &Queue{},
+						RunAs: &JobRunAs{
+							UserName: "user@mail.com",
+						},
```

Comment:
I thought RunAs is not included in get requests?

Reply:
Just to confirm @dvkuznietsov what was the conclusion on these comments? I am guessing these were discussed offline?